ci-operator/jobs/openshift/assisted-test-infra/openshift-assisted-test-infra-master-presubmits.yaml

```diff
@@ -2345,6 +2345,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-single-node-with-worker-live-iso,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Change to
alwayd_run: false
Otherwise it will run for each PR

Reply:
Done.